zero_bin/rpc/src/jerigon.rs

```diff
@@ -35,16 +41,55 @@ where
             "debug_traceBlockByNumber".into(),
             (target_block_id, json!({"tracer": "zeroTracer"})),
         )
-        .await?;
+        .await?
+        .into_iter()
+        .map(|ztr| ztr.result)
+        .collect::<Vec<_>>();
 
     // Grab block witness info (packed as combined trie pre-images)
-
     let block_witness = cached_provider
         .get_provider()
         .await?
         .raw_request::<_, String>("eth_getWitness".into(), vec![target_block_id])
         .await?;
 
+    // // This is wasteful
+    let block = cached_provider
+        .get_block(target_block_id, BlockTransactionsKind::Full)
+        .await?;
+
+    let (_code_db, txns_info) =
+        native::process_transactions(&block, cached_provider.get_provider().await?.deref()).await?;
```

Comment:
as shared offline, we shouldn't have to do this because:

- this is wasteful, we already have the txninfo needed from jerigon payloads
- this induces edges case that are breaking jerigon completeness

Reply:
Fixed.